src/site/content/en/blog/coop-coep/index.md

```diff
@@ -0,0 +1,296 @@
+the same browsing context group. You can use this API to determine whether you 
+have successfully isolated the browsing context group and gained access to 
+powerful features like `performance.measureMemory`.
+
+{% Aside 'caution' %}
+[`self.crossOriginIsolated`](https://www.chromestatus.com/feature/5953286387531776)
+is still under development and not available yet in Chrome as of version 83.
+{% endAside %}
+
+### Debug issues using Chrome DevTools
+
+For resources that are rendered on the screen such as images, it's fairly easy 
+to detect COEP issues because the request will be blocked and the user 
+experience will be visually broken. However, for resources that don't 
+necessarily have a visual impact, such as scripts or styles, COEP issues might 
+go unnoticed. For those, we recommend using the DevTools Network panel. If 
+there's an issue with COEP, you should see 
+`(blocked:NotSameOriginAfterDefaultedToSameOriginByCoep)` in the "Status" 
+column.  
+![Debugging COEP in DevTools 1](devtools1.png)
+
+You can then click the entry to see more details.   
+![Debugging COEP in DevTools 2](devtools2.png)
+
+{% Aside %}
+Note: While COEP debugging is already available, COOP debugging in Chrome 
+DevTools is still [being worked 
+on](https://bugs.chromium.org/p/chromium/issues/detail?id=1051466).
+{% endAside %}
+
+### Observe issues using the Reporting API
+
+The [Reporting 
+API](https://developers.google.com/web/updates/2018/09/reportingapi) is another 
+mechanism through which you can detect various issues. You can configure the 
+Reporting API to instruct your users' browser to send a report whenever COEP 
+blocks the loading of a resource. Chrome supports the `Report-To` header since 
+version 69 for a variety of uses including COEP.
+
+{% Aside %}
+The Reporting API is undergoing transition to [a new 
+](https://w3c.github.io/reporting/)version. Chrome is planning to release it 
+soon, but will leave the older API in place simultaneously for some time. 
+Firefox is also [considering the new 
+API](https://bugzilla.mozilla.org/show_bug.cgi?id=1620573). You may want to use 
+both APIs during the transition.
+{% endAside %}
+
+To specify where the browser should send the reports, append the `Report-To`
+HTTP header to any document that is served with a COEP HTTP header. The
+`Report-To` header also supports a few extra parameters to configure the
+reports. For example:
+
+```http
+Report-To: { group: 'coep_rollout_1', max_age: 86400, endpoints: [{ url: 'https://first-party-test.glitch.me/report'}]}
+```
+
+The parameters object has three properties:
+
+#### `group`
+
+This allows you to name your various reporting endpoints. Use these names to
+direct a subset of your reports. For instance, in the
+`Cross-Origin-Embedder-Policy` directive you can specify the relevant endpoint
+by providing the group name to `report-to=`. For example:
+
+```http
+Cross-Origin-Embedder-Policy: require-corp; report-to="coep_rollout_1"
+```
+
```

Comment:
```suggestion
When the browser encounters this, it will cross-reference the `report-to` value
with the `group` property on the `Report-To` header to look up the endpoint.
This example cross references on `coep_rollout_1' to find the endpoint
`https://first-party-test.glitch.me/report`.

```